src/Microsoft.AspNetCore.SignalR.Common/Protocol/StreamItemMessage.cs

```diff
@@ -17,4 +17,21 @@ public override string ToString()
             return $"StreamItem {{ {nameof(InvocationId)}: \"{InvocationId}\", {nameof(Item)}: {Item ?? "<<null>>"} }}";
         }
     }
+
+    public class ParamStreamMessage : HubMessage
```

Comment:
Thank about renaming this

Reply:
Ideas from meetings: StreamPayload, StreamDataMessage, DataMessage.
Try StreamDataMessage for now